src/snowflake/snowpark/_internal/ast/utils.py

```diff
@@ -470,17 +470,17 @@ def build_indirect_table_fn_apply(
         **kwargs: Keyword arguments to pass to function.
 
     """
-    expr = with_src_position(ast.apply_expr)
+    expr = with_src_position(ast.apply_expr)  # type: ignore[arg-type] # TODO(SNOW-1491199) # Argument 1 to "with_src_position" has incompatible type "ApplyExpr"; expected "Expr"
     if isinstance(
-        func, (snowflake.snowpark.table_function.TableFunctionCall, Callable)
+        func, (snowflake.snowpark.table_function.TableFunctionCall, Callable)  # type: ignore[arg-type] # TODO(SNOW-1491199) # Argument 2 to "isinstance" has incompatible type "tuple[type[TableFunctionCall], <typing special form>]"; expected "_ClassInfo"
     ):
-        stmt = func._ast_stmt
-        fn_expr = expr.fn.indirect_table_fn_id_ref
+        stmt = func._ast_stmt  # type: ignore[union-attr] # TODO(SNOW-1491199) # Item "str" of "Union[str, list[str], TableFunctionCall, Callable[..., Any]]" has no attribute "_ast_stmt", Item "list[str]" of "Union[str, list[str], TableFunctionCall, Callable[..., Any]]" has no attribute "_ast_stmt", Item "TableFunctionCall" of "Union[str, list[str], TableFunctionCall, Callable[..., Any]]" has no attribute "_ast_stmt", Item "function" of "Union[str, list[str], TableFunctionCall, Callable[..., Any]]" has no attribute "_ast_stmt"
```

Comment:
I assume the comments are machine-generated. Does any particular linter or annotation checker require the full message? Looks pretty cumbersome.

Reply:
Yes, it contains three parts:
1. jira no. to track
2. the type ignore 
3. the error message
It's probably verbose for the error message but I try to capture them so when we fix them later, we can easily know what was the issue before.